dnf-behave-tests/features/steps/common/dnf.py

```diff
@@ -116,6 +119,9 @@ def parse_transaction_table(lines):
         result[action] = set()
 
     table_begin = find_transaction_table_begin(lines)
+    if table_begin is None:
+        raise AssertionError("Transaction table not found.")
```

Comment:
Huge amount of tests is failing on this. Mostly probably in case of "Transaction is empty" check, because it checks for ``RuntimeError`` when calling ``parse_transaction_table`` (and it has to try to parse the transaction table, because it's an error if there is some transaction). Can you please fix it?

Reply:
Sorry about that, I thought I've run the tests against the changes, but I guess I must have changed this afterwards.